src/pubsub/ua_pubsub_networkmessage.c

```diff
@@ -771,6 +771,25 @@ UA_NetworkMessage_decodeHeaders(const UA_ByteString *src, size_t *offset, UA_Net
     return UA_STATUSCODE_GOOD;
 }
 
+UA_StatusCode
+UA_NetworkMessage_decodeHeadersRT(const UA_ByteString *src, size_t *offset, UA_NetworkMessage *dst) {
```

Comment:
I wouldn't forward the offset as a pointer.
You cannot use it afterwards anyway. Because it will point to the middle of the headers.
So not usable for the decoding afterwards.

Reply:
ok